src/network/networkctl.c

```diff
@@ -610,10 +618,18 @@ static int dump_address_labels(sd_netlink *rtnl) {
                 if (r < 0)
                         continue;
 
-                printf("%10s/%-5u %30u\n", pretty, prefixlen, label);
+                r = table_add_cell_full(table, NULL, TABLE_UINT32, &label, (size_t) -1, (size_t) -1, 0, 100, 0);
+                if (r < 0)
+                        return r;
+
+                r = table_add_cell_stringf(table, &cell, "%s/%u", pretty, prefixlen);
+                if (r < 0)
+                        return r;
+
+                (void) table_set_align_percent(table, cell, 100);
         }
 
-        return 0;
+        return table_print(table, NULL);
```

Comment:
This looks like this on my machine
```
$ build/networkctl label
LABEL PREFIX/PREFIXLEN 
    0           ::1/128
    1              ::/0
    2         2002::/16
    3             ::/96
    4 ::ffff:0.0.0.0/96
```
The heading of the second column is left-aligned which looks a bit strange. Maybe make it right-aligned like the values?